content/feed.xml

```diff
@@ -3,11 +3,58 @@
   <channel>
     <title></title>
     <description></description>
-    <link>//</link>
-    <atom:link href="//feed.xml" rel="self" type="application/rss+xml"/>
-    <pubDate>Sun, 23 Oct 2016 09:36:43 -0400</pubDate>
-    <lastBuildDate>Sun, 23 Oct 2016 09:36:43 -0400</lastBuildDate>
-    <generator>Jekyll v3.2.1</generator>
+    <link>http://localhost:4000//</link>
+    <atom:link href="http://localhost:4000//feed.xml" rel="self" type="application/rss+xml"/>
```

Comment:
This seems like a bug in the content/feed.xml.  Can you go back and fix this?  Otherwise, it looks great.